src/calliope_pathways/math/pathways.yaml

```diff
@@ -198,6 +198,20 @@ constraints:
       final_step:
         - expression: get_val_at_index(timesteps=-1)
 
+  limit_flow_cap_new_max_rate:
```

Comment:
This constraint is problematic since it will disallow the installment of technologies that do not have initial capacity at 0. The solution other models use is to add a "seed" value that sets an allowed max initial install.

See here: https://temoacloud.com/temoaproject/Documentation.html#temoa_rules.GrowthRateConstraint_rule

Reply:
I catch this below (`where: investsteps=get_val_at_index(investsteps=0) AND flow_cap_initial > 0`). If there is no initial flow cap, there is no constraint in the first timestep.